### PR TITLE
orbitron: init at 20110526

### PR DESCRIPTION
**Pull request by leenaars**
###### Motivation for this change

A very useful font which I needed and thought I'd package so everyone can use it.
###### Things done
- [ ] Tested using sandboxing
  ([nix.useChroot](http://nixos.org/nixos/manual/options.html#opt-nix.useChroot) on NixOS,
    or option `build-use-chroot` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
  - [x] NixOS
  - [ ] OS X
  - [x] Linux
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/data/fonts/orbitron/default.nix [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    ttfdir=$out/share/fonts/ttf/orbitron
+    mkdir -p $otfdir $ttfdir
+    cp -v Orbitron*.otf $otfdir 
+    cp -v Orbitron*.ttf $ttfdir
+  '';
+
+  meta = with lib; {
+    homepage = "https://www.theleagueofmoveabletype.com/orbitron";
+    downloadPage = "https://www.theleagueofmoveabletype.com/orbitron/download";
+    description = ''
```
Comment (joachifm) [inline pkgs/data/fonts/orbitron/default.nix]: The `longDescription` attribute is more appropriate for long-form descriptions; `description` is for short descriptions only.

File: pkgs/data/fonts/orbitron/default.nix [hunk 2]
```diff
@@ -0,0 +1,46 @@
+{ stdenv, lib, fetchFromGitHub }:
```
Comment (joachifm) [inline pkgs/data/fonts/orbitron/default.nix]: `lib` is typically accessed via `stdenv.lib`.

**Comment by leenaars** [comment]
Hi @joachifm, thanks for the review. I made the suggested edits. And did the same to iosevka font, which I had used as a template (did so in a separate commit).

File: pkgs/data/fonts/orbitron/default.nix [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    mkdir -p $otfdir $ttfdir
+    cp -v Orbitron*.otf $otfdir
+    cp -v Orbitron*.ttf $ttfdir
+  '';
+
+  meta = with stdenv.lib; {
+    homepage = "https://www.theleagueofmoveabletype.com/orbitron";
+    downloadPage = "https://www.theleagueofmoveabletype.com/orbitron/download";
+    description = ''
+     Orbitron is a geometric sans-serif typeface intended for display
```
Comment (joachifm) [inline pkgs/data/fonts/orbitron/default.nix]: This description is still way too long. What about simply "A geometric sans-serif font" or something to that effect but not very much longer (note that descriptions shall not contain the name of the package or trailing punctuation).

**Comment by leenaars** [comment]
Hi @joachifm, I've made the description to more dense. Size is now about average length within Nixpkgs I think ... Okay like this?

**Comment by joachifm** [comment]
Thank you

**Pull request merged**